src/marqo/tensor_search/api.py

```diff
@@ -191,6 +193,29 @@ def memory():
     return memory_profiler.get_memory_profile()
 
 
+@app.post('/validate/{index_name}')
+@utils.enable_ops_api()
+def schema_validation(index_name: str, settings_object: str):
```

Comment:
Why are we reading the object as an `str` and `json.loads()` it later? Can we read it as a dictionary here?

Reply:
you to move `json.loads` here?